extra-x11/xf86-video-intel/autobuild/defines

```diff
@@ -14,4 +14,10 @@ AUTOTOOLS_AFTER__I486="${AUTOTOOLS_AFTER__RETRO}"
 
 AB_FLAGS_RRO=0
 AB_FLAGS_NOW=0
+
+# Fedora: This package causes LTO to thrash sucking up enormous amounts of VM.
```

Comment:
Please test and see if this actually applies to us.

Reply:
I've test LTO and unfortunately we are also unable to turn it on. So I copied this explanation from Fedora.